### PR TITLE
fix the doc of inverse, dot, cholesky

### PR DESCRIPTION
**Pull request by ForFishes**
### PR types

Others
### PR changes

Docs
### Describe

fix the doc of cholesky

### CODE REVIEW

File: python/paddle/tensor/linalg.py [hunk 1]
```diff
@@ -453,20 +453,20 @@ def dist(x, y, p=2):
...
 
     This operator calculates inner product for vectors.
    
     .. note::
        Only support 1-d Tensor(vector).
 
     Parameters:
-        x(Variable): 1-D ``Tensor`` or ``LoDTensor``. Its datatype should be ``float32``, ``float64``, ``int32``, ``int64``
-        y(Variable): 1-D ``Tensor`` or ``LoDTensor``. Its datatype soulde be ``float32``, ``float64``, ``int32``, ``int64``
+        x(Variable): 1-D ``Tensor``. Its datatype should be ``float32``, ``float64``, ``int32``, ``int64``
```
Comment (zhiqiu) [inline python/paddle/tensor/linalg.py]: Variable -> Tensor

File: python/paddle/tensor/linalg.py [hunk 2]
```diff
@@ -680,19 +682,18 @@ def cholesky(x, upper=False):
...
+            x_data = np.matmul(a, a_t) + 1e-03
+            x = paddle.imperative.to_variable(x_data)
+            out = paddle.cholesky(x, upper=False)
+            print(out.numpy())
+            # [[1.190523   0.         0.        ]
+            #  [0.9906703  0.27676893 0.        ]
+            #  [1.25450498 0.05600871 0.06400121]]
 
     """
     check_variable_and_dtype(x, 'dtype', ['float32', 'float64'], 'cholesky')
```
Comment (zhiqiu) [inline python/paddle/tensor/linalg.py:695]: Please use `core.ops. cholesky` in dygraph mode.

File: python/paddle/tensor/linalg.py (hunk 1 above)
Comment (ForFishes) [reply python/paddle/tensor/linalg.py]: ok, done.

File: python/paddle/tensor/linalg.py (hunk 2 above)
Comment (ForFishes) [reply python/paddle/tensor/linalg.py:695]: ok, fixed it.

**Pull request merged**